src/multibody/geometry.hpp

```diff
@@ -274,100 +268,61 @@ struct GeometryObject
 
     GeometryModel(const se3::Model & model)
       : model(model)
-      , ncollisions(0)
-      , nvisuals(0)
-      , collision_objects()
-      , visual_objects()
+      , ngeoms(0)
+      , geometry_objects()
       , innerObjects()
       , outerObjects()
     {}
 
     ~GeometryModel() {};
 
-
     /**
-     * @brief      Add a geometry object whose type is COLLISION to a GeometryModel
+     * @brief      Add a geometry object of a given type to a GeometryModel
      *
      * @param[in]  parent     Index of the parent joint
      * @param[in]  co         The actual fcl CollisionObject
      * @param[in]  placement  The relative placement regarding to the parent frame
      * @param[in]  geom_name  The name of the Geometry Object
      * @param[in]  mesh_path  The absolute path to the mesh
+     * @param[in]  type       The type of the GeometryObject
```

Comment:
As we now have two instances of GeometryModel (one for visual, and one for collision), I totally agree.
(note for me : TODO )